internal/controller/controllers/cluster_controller.go

```diff
@@ -58,15 +69,33 @@ func (r *ClusterReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		if k8serrors.IsNotFound(err) {
 			return ctrl.Result{}, nil
 		}
-		return ctrl.Result{}, errors.Wrap(err, "failed to get cluster data")
+		r.Log.Info("failed to get resource %s", req.NamespacedName)
```

Comment:
This is info on purpose (and not capitalized as the other log messages)?

Reply:
Will fix to error log as well.